src/systems/user_commands/UserCommands.cc

```diff
@@ -542,9 +542,9 @@ bool CreateCommand::Execute()
   }
   else if (isLight && isRoot)
   {
-    auto light = root.Light();
-    light->SetName(desiredName);
-    entity = this->iface->creator->CreateEntities(light);
+    auto light = *root.Light();
```

Comment:
Hard to tell, does this cause a copy where there wasn't one before, or is `root.Light()` returning a reference?

Reply:
Yeah the idea is to make a copy, because `Root::Light()` returns a const pointer. I think this is only a problem on Windows. As far as I can tell, that has been returning a const pointer since it was implemented and didn't cause issues on Linux or macOS. That line was introduced in #572, before we enabled Windows CI.